controllers/auth.controller.js

```diff
@@ -0,0 +1,53 @@
+const authService = require("../services/auth.service");
+require("dotenv").config();
+
+const registerUser = async (req, res) => {
+
+  if (JSON.stringify(req.body)==="{}") {
+    return res.send({ status: 400, message: "Request body is empty" });
+  }
+  try {
+    
+    const response = await authService.registerUser(req.body);
+
+
+    if (response.status==201) {
+      res.cookie("jsontoken", response.message, { httpOnly: true });
```

Comment:
Shouldn't the name of the cookie be "jwttoken" or just "jwt" ?

Reply:
I changed this name to find a bug but forget to rename this as **jwt** again. I have renamed it again as **jwt**